api/optimeet/groups/views.py

```diff
@@ -32,5 +35,39 @@ def group_detail(request, group_id):
     return Response(serializer.data)
     
 
+##add user to group
+@csrf_exempt
+@api_view(['POST'])
+def add_users_to_group(request, group_id):
+    try:
+        group = models.Group.objects.get(pk=group_id)
+    except models.Group.DoesNotExist:
+        raise Http404  
+    
+    groupdictionary = {
+    "group_id": group_id,
+    }
+    groupdictionary.update(request.data)
+    serializer = serializers.UserGroupSerializer(data = groupdictionary)
```

Comment:
I found another way to do this that would be a bit cleaner using the Serializer. We'll pass down the `group_id` to the serializer using its `context` argument:
```
serializer = serializers.UserGroupSerializer(data = request.data, context={'group_id': group_id})
```
then in `UserGroupSerializer` we can add a `create` function that will add the context to the data:
```
    def create(self, validated_data):
        validated_data['group_id'] = self.context['group_id']
        return super(UserGroupSerializer, self).create(validated_data)
```
last thing we'd have to change is giving `group_id` a `default=""` in the `UserGroup` model, since `group_id` won't be in data until after the `create` function, and Django will complain about it if it doesn't have a value.